manticore/native/cpu/abstractcpu.py

```diff
@@ -1009,6 +1021,7 @@ def execute(self):
         # FIXME (theo) why just return here?
         # hook changed PC, so we trust that there is nothing more to do
         if insn.address != self.PC:
+            self._last_executed_pc = self.PC
```

Comment:
Hmmmm. I don't entirely remember when this condition hits... I think I need to take a closer look at this to determine why `self.last_executed_pc` isn't set to `curpc`

Reply:
See this commit for the change to `insn.address` https://github.com/trailofbits/manticore/pull/2475/commits/db06f2ca149bc1e2dd3b8f977af4932ffc281c7c